docs/how-to/delete-an-ensembler/delete-an-ensembler.md

```diff
@@ -0,0 +1,18 @@
+# Deleting an Ensembler without any related entity
+
+An Ensembler only could be deleted if there are no active ensembling jobs / router version that use the ensembler. This page show the deletion of ensembler **without any related entity** (router version and ensembling job)
```

Comment:
Nit on some typos:
* `ensembling jobs / router version` -> `ensembling jobs / router versions`
* `This page show` -> `This page shows`